woocommerce/woocommerce
# Fix e2e tests on WordPress 5.7

### All Submissions:






### Changes proposed in this Pull Request:
* In the Docker WordPress 5.7 images, a change was made so that the `wp-config` file is no longer generated statically but instead continues to use the environmental variables. 
* This includes the DB variables in the CLI service so that the site can get properly set up.
* Note: WooCommerce's Travis e2e tests run against WP 5.5.1 right now.
* Props to @youknowriad in https://github.com/WordPress/gutenberg/pull/29770 for the hints.

### How to test the changes in this Pull Request:

1. Follow [documentation](https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e) on setting up local environment for e2e tests.
2. Before running `npm run docker:up`, first set `export WP_VERSION=5.7.0`
3. Run e2e tests using `npm run test:e2e` and verify it works.

### Other information:


### Changelog entry

N/A
